main.cpp

```diff
@@ -1010,7 +901,375 @@ Part 1e - Step 19: Request a review
+    3) solos a given track
+*/
+struct DigitalAudioWorkstation
+{
+    //5 properties:
+    //    1) volume value (float)
+    float volumeValue = 3.5f;
+    //    2) number of tracks (int)
+    int numberOfTracks = 1;
+    //    3) panning value (double)
+    double panValue = 0.0;
+    //    4) type of track (std::string)
+    std::string trackType = "midi";
+    //    5) cost for the software (double)
+    double softwareCost = 299.99;
+
+    struct TrackSlot
+    {
+        bool isTrackPaused = true;
+        std::string actualTimelinePoint = "00h:00m:00s";
+        double gainValue = 1.0;
+        double skipTimeValue = 0.5;
+        bool isIsolated = false;
+
+        void setLoop(std::string startTime = "00h::01m::34s", std::string endTime = "00h::01m::40s"); //play a track from a given list and given the index and selecting a given time. It returns the index of the current playing track
+        void recordTrack(std::string startTime = "00h::00m::00s");
+        double changeGain(double actualGainValue, double addingValue = 0.5); // change the actualVolume by adding an addingValue. It returns the updated volume. 
+    };
+    //3 things it can do:
+    //    1) add a new track
+    void addNewTrack(int trackType, TrackSlot trackSlot); // trackType could range from 0 to whatever number to select the type of track (audio, midi, ecc..)
+    //    2) increase volume of tracks
+    float increaseVolume(int newVolumeValue); // returns the updated volumeValue
+    //    3) solos a given track
+    bool soloTrack(int trackNumber, TrackSlot trackSlot); // uses the track number as parameter to isolate it. It returns true/false if it has been isolated
+};
+
+/*
+Thing 3) Train station
+5 properties:
+    1) number of rails (int)
+    2) number of trains per day (int)
+    3) travelers rate (double)
+    4) number of intercity trains (int)
+    5) number of national trains (int)
+3 things it can do:
+    1) check trains' arrivals
+    2) announce delays
+    3) control travelers access
+*/
+struct TrainStation
+{
+    //5 properties:
+    //    1) number of rails (int)
+    int numberOfRails = 20;
+    //    2) number of trains per day (int)
+    int trainsPerDay = 200; 
+    //    3) travelers rate (double)
+    double travelersRate = 80.2;
+    //    4) number of intercity trains (int)
+    int numberOfIntercityTrains = 10;
+    //    5) number of national trains (int)
+    int numberOfNationalTrains = 10;
+
+    struct Train
+    {
+        int numberOfWagons = 12;
+        int numberOfSeats = 120;
+        int trainId = 123;
+        bool hasArrived = false;
+        float fuelPercentage = 100.0f;
+
+        void leaveStation(float fuelPercentage, bool setAutoPilot = false); // Function to indicate that the train has to leave the station. It accepts fuelPercetnage parameter to check that it can actually leave and if it need to turn the Autopilot on. 
+        double DayMilesTraveled(std::string day); // return the mailes traveled in a given day
```

Comment:
This is a bad function name.  The function name does not match the description.
`computeMilesTraveledPerDay` is a much more accurate function name.
Also, the coding style does not conform to the course coding standard, which is detailed in the Readme.MD. 
- [ ] revise
- [ ] review the course coding standard from the Readme.MD